docs/conf.py

```diff
@@ -172,7 +172,7 @@
 #html_file_suffix = None
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'pep8doc'
+htmlhelp_basename = 'pep8'
```

Comment:
it saves me from keeping a private branch with the fix. The `doc` suffix seems like a bad default anyhow.